modules/ROOT/pages/use-tags-in-visualizer.adoc

```diff
@@ -22,7 +22,7 @@ In the following example, the tag `project-b` was added through properties and t
 image::properties-tags.png[]
 
 [[set-tag-ui]]
-== Setting the Tag Through the s UI
+== Setting the Tag Through the UI
```

Comment:
Which tag? Perhaps "Setting a Tag Through the UI"?
Also, for translators, would "by using" be more comprehensible than "through"?

Reply:
```suggestion
== Set Tags Using the UI
```